source/connect-to-mongo/create-a-mongo-client.txt

```diff
@@ -0,0 +1,122 @@
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies the URI as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+
+        .. TODO: uncomment out when java-rs-auth page made
+        .. For more information about the ``authSource`` connection option, see
+        .. :ref:`java-rs-auth`. 
+
+   * - ``host[:port]``
+
+     - Required. The host and port number where MongoDB is running. If you don't
+       include a port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. 
+        ..    TODO: uncomment out when connection options page made
+        ..    See :ref:`java-rs-connection-options` for a full description of
+        ..    these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string>` in the
+MongoDB Server documentation.
+
+MongoClient
+-----------
+
+To create a connection to MongoDB, pass a connection URI as a string to the
+``MongoClient`` constructor. In the 
+following example, the driver uses a sample connection URI to connect to a MongoDB 
+instance on port ``27017`` of ``localhost``: 
+
+.. code-block:: java
+
+    import com.mongodb.reactivestreams.client.MongoClients;
+    import com.mongodb.reactivestreams.client.MongoClient;
+
+    public class MongoConnection {
+        public static void main(String[] args) {
+            String uri = "mongodb://localhost:27017/";
+            MongoClient client = MongoClients.create(uri);
+        }
+    }
+
+.. tip:: Reusing Your Client
+
+   Because each ``MongoClient`` instance represents a pool of connections to the
+   database, most applications require only a single instance of
+   ``MongoClient``, even across multiple requests. 
+.. TODO: comment out when java-rs-faq page is created
+.. To learn more about connection pools, see the :ref:`FAQ <java-rs-faq>` page.
+
+API Documentation
+-----------------
+
+To learn more about creating a ``MongoClient`` instance in the {+driver-short+},
+see the following API documentation:
+
+- `MongoClient <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/MongoClient.html>`__ 
```

Comment:
```suggestion
- `MongoClient <https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/MongoClients.html>`__ 
```

We should link to `MongoClients` (note `s` at the end), because the `MongoClient` interface documents nothing about creating objects of this type.

Reply:
✅ Resolved.